<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPemeriksaan</class>
 <widget class="QWidget" name="FormPemeriksaan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRUD Pemeriksaan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background:#F5F7FA}QLineEdit,QTextEdit,QComboBox{background:white;border:2px solid #E8E8E8;border-radius:8px;padding:10px;color:#2C3E50}QLineEdit:focus,QTextEdit:focus,QComboBox:focus{border-color:#2E86DE}QPushButton{background:#2E86DE;color:white;border:none;border-radius:8px;padding:12px;font-weight:bold}QPushButton:hover{background:#54A0FF}QPushButton:pressed{background:#1E6FBE}QTableWidget{background:white;border:2px solid #E8E8E8;border-radius:8px;gridline-color:#E8E8E8}QHeaderView::section{background:#2E86DE;color:white;padding:8px;border:none;font-weight:bold}</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <!-- SIDEBAR -->
    <widget class="QFrame" name="frameSidebar">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background:#2E86DE;border:none}</string>
     </property>
     <layout class="QVBoxLayout" name="sidebarLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <!-- Logo Header -->
       <widget class="QFrame" name="frameLogoHeader">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{background:#1E6FBE;border:none}</string>
        </property>
        <layout class="QVBoxLayout" name="logoLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lblLogo">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:white;</string>
           </property>
           <property name="text">
            <string>üè• Klinik</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSubtitle">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#D5E8FF;</string>
           </property>
           <property name="text">
            <string>Sistem Manajemen</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Menu Items -->
       <widget class="QFrame" name="frameMenu">
        <property name="styleSheet">
         <string notr="true">QFrame{background:transparent;border:none}</string>
        </property>
        <layout class="QVBoxLayout" name="menuLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnMenuPemeriksaan">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{background:#1E6FBE;color:white;border:none;border-radius:8px;padding:12px;text-align:left;padding-left:15px}QPushButton:hover{background:#54A0FF}QPushButton:pressed{background:#1560A8}</string>
           </property>
           <property name="text">
            <string>üìã Pemeriksaan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMenuProfil">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{background:transparent;color:white;border:none;border-radius:8px;padding:12px;text-align:left;padding-left:15px}QPushButton:hover{background:rgba(255,255,255,0.1)}QPushButton:pressed{background:rgba(255,255,255,0.2)}</string>
           </property>
           <property name="text">
            <string>üë§ Profil</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <!-- User Info Footer -->
       <widget class="QFrame" name="frameUserInfo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{background:rgba(255,255,255,0.1);border:none;border-radius:8px;margin:15px}</string>
        </property>
        <layout class="QVBoxLayout" name="userInfoLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="lblUserName">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:white;</string>
           </property>
           <property name="text">
            <string>Dr. Ahmad</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblUserRole">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#D5E8FF;</string>
           </property>
           <property name="text">
            <string>Dokter Umum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- CONTENT AREA -->
    <layout class="QHBoxLayout" name="contentLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QFrame" name="frameForm">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{background:white;border-radius:16px}</string>
       </property>
       <layout class="QVBoxLayout" name="formLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="lblJudul">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>üìã Form Pemeriksaan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollContent">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>397</width>
             <height>624</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="scrollLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="lblPendaftaran">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:#2E86DE;</string>
              </property>
              <property name="text">
               <string>1 Data Pendaftaran</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtTglLahir">
              <property name="text">
               <string>Id Pendaftaran</string>
              </property>
              <property name="placeholderText">
               <string>Tanggal Lahir (DD/MM/YYYY)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtNIK">
              <property name="placeholderText">
               <string>NIK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtNama">
              <property name="placeholderText">
               <string>Nama Lengkap</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="txtKeluhan">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Keluhan / Gejala</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblDiagnosa">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:#2E86DE;margin-top:6px;</string>
              </property>
              <property name="text">
               <string>2 Diagnosa &amp; Resep</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtDiagnosa">
              <property name="placeholderText">
               <string>Diagnosa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="txtResep">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Resep Obat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblPembayaran">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:#2E86DE;margin-top:6px;</string>
              </property>
              <property name="text">
               <string>3 Pembayaran</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtBiayaDokter">
              <property name="placeholderText">
               <string>Biaya Dokter (Rp)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtBiayaObat">
              <property name="placeholderText">
               <string>Biaya Obat (Rp)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frameTotal">
              <property name="styleSheet">
               <string notr="true">QFrame{background:#EBF5FF;border-radius:8px;padding:10px}</string>
              </property>
              <layout class="QHBoxLayout" name="layoutTotal">
               <item>
                <widget class="QLabel" name="lblTotalText">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>üí∞ Total:</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spacerTotal">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="lblTotal">
                 <property name="font">
                  <font>
                   <pointsize>13</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:#2E86DE;</string>
                 </property>
                 <property name="text">
                  <string>Rp 0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutButton">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnTambah">
            <property name="styleSheet">
             <string notr="true">QPushButton{background:#27AE60;color:white}QPushButton:hover{background:#2ECC71}QPushButton:pressed{background:#229954}</string>
            </property>
            <property name="text">
             <string>‚ûï Tambah</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdate">
            <property name="styleSheet">
             <string notr="true">QPushButton{background:#F39C12;color:white}QPushButton:hover{background:#F5B041}QPushButton:pressed{background:#D68910}</string>
            </property>
            <property name="text">
             <string>‚úèÔ∏è Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnHapus">
            <property name="styleSheet">
             <string notr="true">QPushButton{background:#E74C3C;color:white}QPushButton:hover{background:#EC7063}QPushButton:pressed{background:#C0392B}</string>
            </property>
            <property name="text">
             <string>üóëÔ∏è Hapus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBersih">
            <property name="styleSheet">
             <string notr="true">QPushButton{background:#95A5A6;color:white}QPushButton:hover{background:#ABB2B9}QPushButton:pressed{background:#7F8C8D}</string>
            </property>
            <property name="text">
             <string>üîÑ Bersih</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameTable">
       <property name="styleSheet">
        <string notr="true">QFrame{background:white;border-radius:16px}</string>
       </property>
       <widget class="QLabel" name="lblData">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>220</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>üìä Data Pemeriksaan</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtCari">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>55</y>
          <width>521</width>
          <height>40</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>üîç Cari berdasarkan nama atau NIK...</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tablePemeriksaan">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>521</width>
          <height>530</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NIK</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nama</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Diagnosa</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>